react-ui/src/redux/reducers/spotsReducer.js

```diff
@@ -1,13 +1,14 @@
 import {
     FETCH_SPOTS_REQUEST, FETCH_SPOTS_SUCCESS, FETCH_SPOTS_FAILURE,
-    FETCH_SPOTS_CONSTANTS_SUCCESS, FETCH_SPOTS_CONSTANTS_FAILURE
+    FETCH_SPOTS_CONSTANTS_SUCCESS, FETCH_SPOTS_CONSTANTS_FAILURE,
+    FETCH_SPOT_DETAILS_REQUEST, FETCH_SPOT_DETAILS_SUCCESS, FETCH_SPOT_DETAILS_FAILURE,
 } from '../actions/types';
 import { SUCCESS, BAD_SPOTS_FETCH } from '../errorMessages'
 
 const initialState = {
     fetchingSpots: false,
-    spotsFetched: false,
-    constantsFetched: false,
+    spotsFetched: false,     // would like to remove this if no one is using
+    constantsFetched: false, // would like to remove this if no one is using
```

Comment:
spotsFetched is checked for load spinner on spotlight page, but so far haven't used constantsFetched !

Reply:
cool, I will leave both in for now